services/galley/src/Galley/API/Action.hs

```diff
@@ -897,7 +897,9 @@ notifyConversationAction tag quid notifyOrigDomain con lconv targets action = do
         -- because quid's backend will update local state and notify its users
         -- itself using the ConversationUpdate returned by this function
         if notifyOrigDomain || tDomain ruids /= qDomain quid
-          then fedQueueClient @'OnConversationUpdatedTag update $> Nothing
+          then do
+            env <- ask
+            fedQueueClientBundle (toBundle @'OnConversationUpdatedTag env update) $> Nothing
```

Comment:
`OnConversationUpdatedTag` is the fed end-point in its most recent (locally available) version, right?  And for older versions of the same end-point, you need different tags?  How does `toBundle` know where to find those older tags?

If I'm right and this is a problem, maybe using a [`Dynamic`](https://hoogle.zinfra.io/file/nix/store/5x7yh702gmyiwhbiq1qbybknyk1fl4ys-ghc-9.4.8-doc/share/doc/ghc/html/libraries/base-4.17.2.1/Data-Dynamic.html) list in the bundle is part of the answer?  But then you still need to figure out how to guess the tags that are in there.

If this makes no sense, I'll think about it more tomorrow.  :)